src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write code here
+const li = document.querySelectorAll('li');
+
+li.forEach((item) => {
+  const span = document.createElement('span');
+
+  span.textContent = item.firstChild.textContent.trim();
```

Comment:
Ensure that `item.firstChild` is indeed a text node. If the `li` element contains other elements, `firstChild` might not be the text node you expect. Consider checking the node type or using `item.childNodes` to find the correct text node.